src/Resources/config/doctrine/Sylius.Customer.orm.xml

```diff
@@ -27,5 +27,15 @@
         </inverse-join-columns>
       </join-table>
     </many-to-many>
+    <many-to-many field="groups" target-entity="AppBundle\Entity\Sylius\CustomerGroup">
```

Comment:
This mapping is [already managed by the superclass](https://github.com/Sylius/Sylius/blob/bbb6233c1d180fce360554254b902a164be4ce00/src/Sylius/Bundle/CustomerBundle/Resources/config/doctrine/model/Customer.orm.xml#L44-L46)

Reply:
it's not same relationship